cmd/watch_repo/style.go

```diff
@@ -0,0 +1,41 @@
+package main
```

Comment:
Can any of these things be made into a package to be reused by other utils in the repo?